src/seedu/addressbook/AddressBook.java

```diff
@@ -763,7 +763,7 @@ private static void createFileIfMissing(String filePath) {
     private static void savePersonsToFile(ArrayList<String[]> persons, String filePath) {
         final ArrayList<String> linesToWrite = encodePersonsToStrings(persons);
         try {
-            Files.write(Paths.get(storageFilePath), linesToWrite);
+            Files.write(Paths.get(filePath), linesToWrite);
```

Comment:
Nice job catching this 👍 another fix could be omitting `filePath` parameter entirely from the method signature since it is not used in the method anyway